bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties

```diff
@@ -1,56 +1,57 @@
 # binding
 binding.shelly.name = Shelly Binding
-binding.shelly.description = Dieses Binding unterstützt Shelly Geräte, die über WiFi gesteuert werden können.
+binding.shelly.description = Dieses Binding unterst�tzt Shelly Ger�te, die �ber WiFi gesteuert werden k�nnen.
```

Comment:
The translation needs to be updated in Crowdin, once the translation is loaded into Crowdin.